avm/ptn/dev-ops/cicd-agents-and-runners/scripts/runPlaceHolderJob.ps1

```diff
@@ -0,0 +1,31 @@
+﻿Param(
```

Comment:
This appears to be a non-ideal naming for a script as it seems to be waiting for the job to start and should hence be named as such.
It would also probably be a good call to add a synopsis to explain the script's intend and what its parameters do :)

Reply:
Resolved

Reply:
The script is named placeholder as in container apps with Azure devops, there needs to be a "placeholder" agent running before deploying other agents

Reply:
Still worst naming ever :P 
Anyhow - in the script itself there does not seem to be a reference to the placeholder agent. In other words: It seems to be 'abstract' enough to be called something that points to what it is doing, if that makes sense.

Reply:
Should be resolved now 👍

Reply:
Love it. Thank you @sebassem 💪